### PR TITLE
disable usage of namespace name

### PR DESCRIPTION
**Pull request by nurali-techie**
To further analyze issue https://github.com/redhat-developer/che-functional-tests/issues/470 we have added logs and disable usage of namespace name.  Below are the changes.

1. Added tests for namespace name with che specific url.
2. Remove usage of namespace name.  Namespace name is only logged and pass around method param but not used in actual logic.

### CODE REVIEW

File: middlewares/osio/osio_che_test.go [hunk 1]
```diff
@@ -52,32 +52,32 @@ var cheCtx = testCheCtx{tables: []testCheData{
...
-		"3000_che_secret",
-	},
-	{
-		// user=22222222-* wants to access its own ns=osio-test-preview-che resuorces
-		"/apis/apps/v1/namespaces/osio-test-preview-che/pods",
-		"22222222-1874-4de5-9c62-602634cb5cc2",
-		"127.0.0.1:9091",
-		"4000_che_secret",
-	},
+	// {
```
Comment (xcoulon) [inline middlewares/osio/osio_che_test.go]: should you remove those commented-out lines?

File: middlewares/osio/osio.go [hunk 2]
```diff
@@ -130,12 +130,12 @@ func (a *OSIOAuth) cacheResolverByID(token string, tokenType TokenType, userID s
 			log.Errorf("Failed to locate cluster token, %v", err)
 			return cacheData{}, err
 		}
-		secretName, err := a.RequestSecretLocation.GetName(namespace.ClusterURL, clusterToken, namespaceName, namespace.Type)
+		secretName, err := a.RequestSecretLocation.GetName(namespace.ClusterURL, clusterToken, namespace.Name, namespace.Type)
```
Comment (xcoulon) [inline middlewares/osio/osio.go]: should you remove the `namespaceName` param from the method if it's unused?

**Comment by nurali-techie** [comment]
@xcoulon thanks for approving.  Yes, it's only for further analysis of issue by adding extra logs.  There will be follow PR after analysis is captured :+1:

**Comment by nurali-techie** [comment]
[test]

**Pull request merged**